### ISSUE TITLE
Bug in relu_prime?

### ISSUE THREAD

**Issue by bremen79** [lijunwei19/CNN-matlab-covid-9-x-ray#1]
Hi, 
the subgradient of the relu is 0 or 1, while it seems that you are returning 0 or the input itself. A bug?

**Comment by lijunwei19** [comment]
yes, I am wrong, I had changed the Relu derivative function. 0 or 1. thanks
